tsl/src/fdw/fdw.c

```diff
@@ -395,6 +406,8 @@ static FdwRoutine timescaledb_fdw_routine = {
 	/* explain/analyze */
 	.ExplainForeignScan = explain_foreign_scan,
 	.ExplainForeignModify = explain_foreign_modify,
+	/* Support functions for join push-down */
+	.GetForeignJoinPaths = ts_get_foreign_join_paths,
```

Comment:
For the first version, are we going with reference tables as normal manually replicated tables? This would mean that this hook is not needed, because it is only called for joins of two foreign tables from the same foreign server. Might make sense to clarify this in comments or even remove it.

Reply:
Good point, it is removed.